tests/queries/0_stateless/02008_aliased_column_distributed_bug.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS click_storage;
+DROP TABLE IF EXISTS click_storage_dst;
+
+CREATE TABLE click_storage ( `PhraseID` UInt64, `PhraseProcessedID` UInt64 ALIAS if(PhraseID > 0, PhraseID, 0) ) ENGINE = MergeTree() ORDER BY tuple();
+
+CREATE TABLE click_storage_dst ( `PhraseID` UInt64, `PhraseProcessedID` UInt64 ) ENGINE = Distributed(test_shard_localhost, currentDatabase(), 'click_storage');
+
+SELECT materialize(PhraseProcessedID)　FROM click_storage_dst;
```

Comment:
We need to test both local plans and remote plans. In order to achieve this, some data has to be inserted. Then we use `settings prefer_localhost_replica = 0` and `settings prefer_localhost_replica = 1`

Reply:
When I added it it broke my solution. But with check of `distributed_depth` and `distributed_group_by_no_merge` that you suggested seems that it works, thanks!

Let's wait for CI.